### ISSUE TITLE
Support Relationship attribute type

### ISSUE THREAD

**Issue by c0c0n3** [smartsdk/ngsi-timeseries-api#383]
### Is your feature request related to a problem? Please describe.
The `Relationship` attribute type is not officially part of the NGSI v2 spec but is nonetheless widely used by FiWare applications in practice. QuantumLeap does not include this type in its NGSI-to-SQL translation map.

### Describe the solution you'd like
Add the `Relationship` type to the NGSI-to-SQL map and save `Relationship` attribute values as plain text. Notice `Relationship` attribute values will **not** be given any specific meaning (e.g. attempting to interpret them as NGSI-LD references), rather they will be opaque strings QuantumLeap stores just like any other text attribute. In particular, they won't be mapped to foreign keys or similar relational constructs.

### Describe alternatives you've considered
Do nothing. After #382 gets fixed, `Relationship` attribute values can be saved to the DB. But this isn't the cleanest solution and it also has the annoying side-effect of the warning below being logged on every attempt to save a `Relationship` attribute value

```
WARNING:translators.crate:'Relationship' is not a supported NGSI type.
Please use any of the following: Array, Boolean, ISO8601, DateTime,
Integer, geo:json, geo:point, Number, Text, StructuredValue. Falling
back to Text.
```

### Additional context
#382

**Comment by c0c0n3** [comment]
### NGSI-LD
During Nov 2020's FiWare TSC, we got a confirmation that NGSI-LD does **not** really support a temporal model w/r/t entity graphs. According to the TSC, all we need to do to claim QL supports the NGSI-LD temporal entity/relationship model is to store entity references as plain NGSI text attributes just like this issues suggests to do for the `Relationship` type of NGSI-v2.

So implementing a relationship type would be quite a leap forward in the mythical quest for NGSI-LD support!

### Implications
This kind of approach has the downside that answering questions about your data over time that involve relationships can be prohibitively expensive which begs the question of what's the practical usefulness of NGSI-LD/temporal?! Notice TSC agreed that client applications will have to reconstruct the (temporal) graph they need from the ref attributes stored in entity rows, so technically we don't have a problem w/r/t being NGSI-LD compliant. But, but... how can clients possibly do that over large datasets through a Web API?! Also this kind of set up basically rules out big data analytics involving relations. Oh well.

**Comment by chicco785** [comment]
this should be fixed by #373
